docs/components/modeler/web-modeler/advanced-modeling/manage-connector-templates.md

```diff
@@ -0,0 +1,103 @@
+---
+id: manage-connector-templates
+title: Manage connector templates
```

Comment:
The last state of the discussion was to capitalize `Connectors`

Reply:
(This is the case in general for this PR. Maybe @christinaausley can clarify)

Reply:
I updated the capitalization in the title, but not in the text; @christinaausley I think it makes sense to merge this first, then update the capitalization in-text in a seperate PR over all occurences in the docs?

Reply:
I am launching a PR shortly to uppercase all instances of Connectors, but I'll go ahead and capitalize those instances here while doing my technical review.